server/repository/src/db_diesel/temperature_breach_config.rs

```diff
@@ -97,24 +100,45 @@ type BoxedLogQuery = temperature_breach_config::BoxedQuery<'static, DBType>;
 fn create_filtered_query(filter: Option<TemperatureBreachConfigFilter>) -> BoxedLogQuery {
     let mut query = temperature_breach_config::table.into_boxed();
 
-    if let Some(filter) = filter {
-        apply_equal_filter!(query, filter.id, temperature_breach_config_dsl::id);
-        apply_equal_filter!(query, filter.r#type, temperature_breach_config_dsl::type_);
+    if let Some(TemperatureBreachConfigFilter {
```

Comment:
This destructuring helps when new filter is added (compiler complains and allows you to add extra field with cmd + .